samples/bluetooth/broadcast_config_tool/src/main.c

```diff
@@ -31,6 +31,10 @@ ZBUS_CHAN_DECLARE(sdu_ref_chan);
 ZBUS_CHAN_DECLARE(le_audio_chan);
 ZBUS_MSG_SUBSCRIBER_DEFINE(le_audio_evt_sub);
 
+/* SD card detection */
+static bool sd_card_not_present;
+static uint8_t frame_value;
```

Comment:
This should only be visible where it is used. Should be moved into `stream_frame_dummy_get_and_send`